components/CollectionCells/examples/CollectionCellsTextExample.m

```diff
@@ -122,7 +122,7 @@ + (NSString *)catalogDescription {
 }
```

Comment:
do we still want to keep the catalogDescription and the catalogIsPrimaryDemo?

Reply:
Probably ok to leave.